src/components/ArticleComments.jsx

```diff
@@ -1,12 +1,16 @@
 import { useEffect, useState } from "react"
-import { fetchArticleComments } from "../../utils/api"
+import { commentOnArticle, fetchArticleComments } from "../../utils/api"
 import Loading from './Loading';
 import Error from './Error';
 import CommentCard from "./CommentCard";
 import getUserData from "../../utils/getUserData"
+import PostCommentCard from "./PostCommentCard";
 
 export default function ArticleComments({ article_id, users }) {
 
+    // Hardcoded in current user, at this level for now. Move upwards later
+    let currentUser = getUserData(users, 'tickle122')
```

Comment:
I would replace this with a hard coded user like `const loggedInUser = 'tickle122'`;
In reality you would not fetch all users to log in, find out which one we are.